steering_committee_membership_guidelines.rst

```diff
@@ -132,3 +132,8 @@ The process of the election consist of the following steps:
    person responsible for arranging the election about that.
 #. Conduct anonymous voting somewhere.
 #. Internally and publicly announce the elected candidate.
+
+The chairperson has the following powers unlike regular members:
+
+* The chairperson has a casting vote when vote deadlocks in `community topics <https://github.com/ansible-community/community-topics/issues>`_ happen (equal numbers of votes for and against a topic.
+* Has a power of veto - the power to unilaterally reject a decision made by the Committee. In this case, the chairperson must explain their motives in a corresponding `community topic <https://github.com/ansible-community/community-topics/issues>`_.
```

Comment:
we discussed this a long time ago, so I don't remember the details, frankly speaking, but, i guess, it was taken by analogy with other projects/public institutions. It can be overwritten (imo, by 3/4 of the SC feels sensible, I'll add this to the other doc now, thanks @felixfontein for suggesting).
I'm not opposed to get rid of the veto entirely if other members agree.

Reply:
Done, PTAL